examples/microViT.py

```diff
@@ -258,11 +258,13 @@ def test_step(self, batch, _):
         num_classes=num_classes,
         attention="scaled_dot_product",
         classifier=Classifier.TOKEN,
+        layer_norm_style="deepnorm",
+        use_rotary_embeddings=False,
     )
     trainer = pl.Trainer(
         gpus=GPUS,
         max_epochs=MAX_EPOCHS,
-        detect_anomaly=True,
```

Comment:
@SeanNaren we discussed this at some point I think, but this flag completely destroys the speed, on a single GPU (so it's not because of communication). May be worth a warning/explanation ?